doc/classes/CodeEdit.xml

```diff
@@ -50,12 +50,20 @@
 			<param index="4" name="icon" type="Resource" default="null" />
 			<param index="5" name="value" type="Variant" default="null" />
 			<param index="6" name="location" type="int" default="1024" />
+			<param index="7" name="is_swizzling_component" type="bool" default="false" />
 			<description>
 				Submits an item to the queue of potential candidates for the autocomplete menu. Call [method update_code_completion_options] to update the list.
 				[param location] indicates location of the option relative to the location of the code completion query. See [enum CodeEdit.CodeCompletionLocation] for how to set this value.
 				[b]Note:[/b] This list will replace all current candidates.
 			</description>
 		</method>
+		<method name="add_code_completion_swizzling_pattern">
+			<return type="void" />
+			<param index="0" name="pattern" type="String" />
+			<description>
+				Adds a swizzling completion pattern (eg. [code]xyzw[/code]).
```

Comment:
```suggestion
				Adds a swizzling completion pattern (e.g. [code]xyzw[/code]).
```
But some of us don't like `i.e.` or `e.g.`,  and have been trying to fully spell things out for clarity:
```suggestion
				Adds a swizzling completion pattern, for example [code]xyzw[/code].
```